cedar-drt/fuzz/fuzz_targets/convert-schema-json-to-cedar.rs

```diff
@@ -58,7 +58,7 @@ fuzz_target!(|src: String| {
             Err(
                 cedar_policy_core::validator::cedar_schema::fmt::ToCedarSchemaSyntaxError::NameCollisions(
                     _,
-                ),
+                ) | cedar_policy_core::validator::cedar_schema::fmt::ToCedarSchemaSyntaxError::UnconvertibleEntityTypeShape(_),
             ) => {
                 // Currently, we ignore name-collisions errors, as JSON schemas encountering name-collisions errors are not supported for conversion to Cedar format; see cedar#1272
```

Comment:
Should we add to this comment and explain why we ignore this class of error? and point to the issue again?